src/net/sourceforge/kolmafia/swingui/panel/ItemListManagePanel.java

```diff
@@ -19,11 +19,11 @@ public ItemListManagePanel(
         addFilterField,
         addRefreshButton);
 
-    ShowDescriptionList<E> elementList = (ShowDescriptionList<E>) this.elementList;
+    ShowDescriptionList<E> elementList = this.scrollComponent;
```

Comment:
Is this correct? I was surprised to see a change of property name referenced.

Reply:
This is because `elementList` was only made to "save" a type casting

`JComponent elementList = this.scrollComponent;`

(see ItemManagePanel)